build-service.py

```diff
@@ -90,10 +90,20 @@ def build_repo(repo, branch, config):
                     config = yaml.load(f, Loader=yaml.Loader)
             except FileNotFoundError:
                 raise MissingConfigFile from FileNotFoundError
-
         image = config["spec"]["template"]["spec"]["containers"][0]["image"]
         dclient.images.build(path=repo_dir, tag=image)
 
+    # deploying using kubernetes python api
+    try:
+        # load kube config from ~/.kube/config
+        kubernetes.config.load_kube_config()
+        # equivalent to "kubectl apply -f [config]"
+        k8s_apps_v1 = kubernetes.client.AppsV1Api()    
+        resp = k8s_apps_v1.create_namespaced_deployment(body=config, namespace="default")
```

Comment:
Bit confused here, so we create a deployment and set it to resp but don't use it.
Any reason for the unused variable?

Reply:
Ah yeah originally the example I went off of used the resp to print out some status info from the deployment. I'll go ahead and remove that variable

Reply:
Also, dont think it's important. But do you know why we are using `default` for the namespace? 
Do we want to have the ability to create different namespaces like `production` for example?

https://kubernetes.io/docs/tasks/administer-cluster/namespaces-walkthrough/

Looks like namespaces allow people to share a Kubernetes cluster.

Reply:
Do we want to make a new namespace for each project? Since this is a shared tenant instead of just for a single project this might help keep stuff separate. Name could be based off of the image name or smth.